src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,412 @@
+            (context.status === 'released' || context.status === 'released to project')) {
+            this.setState({
+                'modal': (
+                    <ConfirmModal handleConfirm={this.handleSave} handleCancel={this.handleModalCancel}
+                        confirmButtonText="Save" cancelButtonText="Cancel" modalTitle="Confirm Save">
+                        You are overwriting a Microscope Configuration Item that was previously shared with public. Are you sure?
+                        <br />Note that you can also clone this display and share the new copy.
+                    </ConfirmModal>
+                )
+            });
+            return;
+        }
+
+        mtc.api.saveMicroscope();
+    }
+
+    statusChangeButton(){
+        const { session, context } = this.props;
+        const { saveLoading, cloneLoading, releaseLoading } = this.state;
+        const editPermission = this.havePermissionToEdit();
+
+        if (!session || !editPermission) return null; // TODO: Remove and implement for anon users. Eventually.
+
+        const btnProps  = {
+            'onSelect'      : this.handleStatusChange,
+            //'onClick'       : context.status === 'released' ? null : this.handleStatusChangeToRelease,
+            'variant'       : context.status === 'released' ? 'outline-dark' : 'info',
+            'disabled'      : releaseLoading,
+            'key'           : 'statuschangebtn',
+            'data-tip'      : "Change the visibility/permissions of this HiGlass Display",
+            'title'         : (
+                <React.Fragment>
+                    <i className={"icon icon-fw icon-" + (releaseLoading ? 'circle-notch fas icon-spin' : 'id-badge far')}/>&nbsp; Manage
+                </React.Fragment>
+            ),
+            'pullRight'     : true
+        };
+
+        return (
+            <DropdownButton {...btnProps}>
+                <StatusMenuItem eventKey="released" context={context}>Visible by Everyone</StatusMenuItem>
+                <StatusMenuItem eventKey="released to project" context={context}>Visible by Network</StatusMenuItem>
+                <StatusMenuItem eventKey="released to lab" context={context}>Visible by Lab</StatusMenuItem>
+                <StatusMenuItem eventKey="draft" context={context}>Private</StatusMenuItem>
+                <Dropdown.Divider />
+                {/* These statuses currently not available.
+                <StatusMenuItem active={context.status === "archived to project"} eventKey="archived to project">Archive to Project</StatusMenuItem>
+                <StatusMenuItem active={context.status === "archived"} eventKey="archived">Archive to Lab</StatusMenuItem>
+                */}
+                <StatusMenuItem eventKey="deleted" context={context}>Delete</StatusMenuItem>
+            </DropdownButton>
+        );
+    }
+
+    saveButton(){
+        const { session, context } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['context' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)